recipes/nupic/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "nupic" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "048dae895ca505dddb46efb8ad062f58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It is unclear if all of these libraries are actually required to build the package. My guess is that most of them can be removed since they are testing requirements or runtime reqs, not build reqs.